index.js

```diff
@@ -26,11 +24,7 @@ var main = function(url, options) {
 	try {
 		options = options || {};
 
-		if (typeof options !== "object") {
-			throw "Expected an object";
-		}
-
-		return options.embed === true ? getEmbed(url) : getPanoID(url);
+		return options.embed ? getEmbed(url) : getPanoID(url);
```

Comment:
I can't remove the condition to check if options is an object because all code of the main function are envolved from try/catch block and also I use the exception in [another package](https://github.com/dorianneto/get-streetview-panorama-id-cli) to validate some conditions.